content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm

```diff
@@ -99,14 +99,16 @@
                             <fieldset>
                                 <legend><h4>$tlang.getString("avail.title") <small>$tlang.getString("avail.uploaded") </small></h4></legend>
                                 <div class="form-group">
-                                    <input type="radio" name="hidden" id="hidden_false" value="false" #if(!$upload_visibility_hidden)checked="checked" #end/>
+                                    <input type="radio" name="hidden" id="hidden_false" value="false" #if(!$upload_visibility_hidden)checked="checked" #end
+                                        onclick="$('#selectNotificationOptions').show();" />
```

Comment:
is there no JS file where you can put things like this ?

Reply:
Possibly. This patch applied cleanly and I noticed there were other onclicks defined in the HTML, so I didn't investigate moving them.